styles.css

```diff
@@ -91,192 +100,218 @@ li {
 .down-icon img {
   border: 1px solid #def1e6;
   border-radius: 50%;
-  width: 30px;
   display: block;
   margin: auto;
 }
 
-.section-title {
-  padding-top: 30px;
-  padding-bottom: 30px;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
+.work-container h2 {
+  color: #172b4d;
   text-align: center;
+  font-size: 2.5rem;
+  margin-top: 1rem;
 }
 
-.card-container {
+hr {
+  background-color: #36b37e;
+  border-radius: 24px;
+  width: 48px;
+  height: 4px;
+  margin: auto;
+  margin-top: 1rem;
+  margin-bottom: 2rem;
+}
+
+.work-grid {
   display: grid;
-  grid-template-columns: 1fr;
-  row-gap: 24px;
+  grid-auto-rows: auto;
+  grid-template-columns: repeat(auto-fill, minmax(min(100%, 13rem), 1fr));
+  gap: 1vw;
+  margin: 2vw;
 }
```

Comment:
Great work @MarcoOquendoC 🤝

- [ ] Just like the previous reviewer suggested. You may please make sure your projects appear in two rows just like in the Figma template provided
- Your design
![your projects](https://user-images.githubusercontent.com/63917725/178123625-2eee368a-92dc-4cfb-b729-542348e4aba1.PNG)

- Figma design
![Figma projects](https://user-images.githubusercontent.com/63917725/178123649-660c60cc-d5d3-484c-a008-ecb207068165.PNG)